gdx/src/com/badlogic/gdx/math/Matrix4.java

```diff
@@ -1124,131 +1134,132 @@ public String toString () {
 	}
 
 	// @off
-	/*JNI
```

Comment:
I think it you will have be to revert ALL changes to the following lines.

Reply:
@dar-dev